examples/guided.py

```diff
@@ -70,124 +71,59 @@ def ask_user_questions():
 		Not until we're satisfied with what we want to install
 		will we continue with the actual installation steps.
 	"""
-	if not archinstall.arguments.get('keyboard-layout', None):
-		archinstall.arguments['keyboard-layout'] = archinstall.select_language()
 
-	# Before continuing, set the preferred keyboard layout/language in the current terminal.
-	# This will just help the user with the next following questions.
-	if len(archinstall.arguments['keyboard-layout']):
-		archinstall.set_keyboard_language(archinstall.arguments['keyboard-layout'])
+	global_menu = GlobalMenu()
```

Comment:
You can omit the `from archinstall.lib.menu.selection_menu import GlobalMenu` and use `archinstall.GlobalMenu` as long as you expose the class in: https://github.com/archlinux/archinstall/blob/577a060e792a82e1e93822c9a16b4093f94c003c/archinstall/__init__.py#L23
This will make the guided example a bit more cleaner. As it serves as an example of how to write an installer not having to dig through the inners of archinstall and learn all the paths to stuff :)

Reply:
@Torxed awesome thanks for the review! 
I've updated the import

Reply:
Great work! We've modified guided a bit in regards to `"!users"` and `"harddrives"` section, could you pull that in to the global menu? I think that would help with the merge conflict against master that is right now.